### DISCUSSION TITLE
Change the countdown display

### DISCUSSION THREAD

**Discussion by kandlerd** [photobooth-app/photobooth-app#171] [Q&A] [answered]
Hi, 
thanks for the great software.

I use the photo booth with a Raspberry 4, a PiCamV2 and hardware buttons. 

Is there a way to display the countdown to taking pictures differently?  So, for example, remove the circle and display the numbers quite large as a countdown?

Thanks and best regards,
Dirk

**Comment by mgineer85** [comment, marked as answer]
No, this kind of individualization is not implemented unfortunately.
At a later point we could add custom css that would allow to modify.
I added your request to https://github.com/photobooth-app/photobooth-app/issues/86 so it will be covered once solved.

**Reply by kandlerd** [reply]
All right, thanks for the reply.

**Reply by mgineer85** [reply]
Latest v2 beta already has bigger countdown. Maybe this is acceptable for you already as intermediate solution.
